server/src/yml-objects/YmlObjectInstance.ts

```diff
@@ -8,7 +8,14 @@ export class YmlObjectInstance extends AbstractYmlObject {
     public domainsLevel2: YmlType;
     public readonly kind = CompletionItemKind.Variable;
 
-    constructor(indentifier: string, uri: string) {
+    constructor(indentifier: string, uri: string, public readonly isLocal) {
         super(indentifier, CompletionItemKind.Variable, uri);
+        this.kindName = isLocal ? 'variable' : 'instance';
+    }
+
+    public setDetail(type: string): void {
+        this.detail = this.isLocal
```

Comment:
Just one `=`? :thinking: And weird comparison if it is one (details compared to being local?).

Reply:
We create `YmlObjectInstance` when we encounter a static instance, a local variable or a function argument.
Here I make the distinction between static instances (not local) by adding `[GLOBAL]` to the detail and the rest.